.travis.yml

```diff
@@ -46,10 +46,9 @@ matrix:
         - PGVER=12
```

Comment:
Can you please add a line in the `CHANGELOG` for this? I guess this will also make our next release one with breaking changes since we're removing endpoints.